engine/apps/webhooks/models/webhook.py

```diff
@@ -337,3 +339,16 @@ class WebhookResponse(models.Model):
     def json(self):
         if self.content:
             return json.loads(self.content)
+
+
+@receiver(post_save, sender=WebhookResponse)
+def webhook_response_post_save(sender, instance, created, *args, **kwargs):
+    if not created:
+        return
+
+    # TODO: find a better way to filter, additional_settings__isnull is a workaround
+    source_alert_receive_channel = instance.webhook.filtered_integrations.filter(
+        additional_settings__isnull=False
+    ).first()
+    if source_alert_receive_channel and hasattr(source_alert_receive_channel.config, "on_webhook_response_created"):
+        source_alert_receive_channel.config.on_webhook_response_created(instance, source_alert_receive_channel)
```

Comment:
Would it possible to add some test for this?

Reply:
done in https://github.com/grafana/oncall/pull/4076/commits/34baeba4adbe7f199dce60a7a48e39e92d93fad8